frontend/src/container/Header/DateTimeSelection/index.tsx

```diff
@@ -206,20 +206,29 @@ const DateTimeSelection = ({
 
 	// this is triggred when we change the routes and based on that we are changing the default options
 	useEffect(() => {
-		const metricsTimeDuration = getLocalStorageKey(
-			LOCAL_STORAGE.METRICS_TIME_IN_DURATION,
-		);
+				
+		const currentStorage = getLocalStorageKey(LOCAL_STORAGE.METRICS_TIME_IN_DURATION);
 
-		if (metricsTimeDuration === null) {
+		if(currentStorage === null){
 			setLocalStorageKey(
 				LOCAL_STORAGE.METRICS_TIME_IN_DURATION,
-				JSON.stringify({}),
-			);
+				JSON.stringify({
+					'/' : '5min',
+					'/application':'5min',
```

Comment:
this is not the scalable solution @Deeprana12 

use route from `lib/routeName`